yt_dlp/extractor/vocaroo.py

```diff
@@ -57,7 +57,7 @@ def _real_extract(self, url):
             'title': '',
             'url': url,
             'ext': 'mp3',
-            'timestamp': float_or_none(resp.getheader('x-bz-upload-timestamp'), scale=1000),
+            'timestamp': float_or_none(resp.get_header('x-bz-upload-timestamp'), scale=1000),
```

Comment:
I think this is preferred
```suggestion
            'timestamp': float_or_none(resp.headers.get('x-bz-upload-timestamp'), scale=1000),
```